docs/docs/ecommerce-tutorial/index.md

```diff
@@ -0,0 +1,524 @@
+---
+title: "Gatsby E-Commerce Tutorial"
+---
+
+# Table of Contents
+1. [Why use Gatsby for an e-commerce site?](#why-use-gatsby-for-an-e-commerce-site)
+2. [Prerequisites](#prerequisites)
+3. [Setting up a Gatsby site](#setting-up-a-gatsby-site)
+4. [Installing Stripe Checkout plugin](#installing-stripe-checkout-plugin)
+5. [Creating a button](#creating-a-button)
+6. [Importing checkout component into homepage](#importing-checkout-component-into-homepage)
+7. [Getting Your Stripe test keys](#getting-your-stripe-test-keys)
+8. [Configuring Stripe in Gatsby](#configuring-stripe-in-gatsby)
+9. [Setting up a serverless function in AWS Lambda](#setting-up-a-serverless-function-in-aws-lambda)
+10. [Setting up a separate repo](#setting-up-a-separate-repo)
+11. [Editing your new repo](#editing-your-new-repo)
+12. [Pushing code to AWS](#pushing-code-to-aws)
+13. [Configuring serverless with your AWS credentials](#configuring-serverless-with-your-aws-credentials)
+14. [Testing Payments](#testing-payments)
+
+
+# Why use Gatsby for an e-commerce site?
+
+In this tutorial, you’ll learn how to use Gatsby to build the UI for a basic e-commerce site that can accept payments, with Stripe as the backend for processing payments. Benefits of using Gatsby for e-commerce sites include the following:
+* Security inherent in static sites
+* Blazing fast performance when your pages are converted from React into static files
+* Easy to host
+
+You can see the working demo hosted here: https://gatsby-ecommerce.netlify.com/
+
+# Prerequisites
+
+* Stripe account: [register for an account here](https://dashboard.stripe.com/register)
+* AWS account (free tier that covers anywhere from several thousand to a million requests per month): [register for an account here](https://aws.amazon.com/free/?sc_channel=PS&sc_campaign=acquisition_US&sc_publisher=google&sc_medium=cloud_computing_b&sc_content=aws_account_e_control_q32016&sc_detail=create%20an%20aws%20account&sc_category=cloud_computing&sc_segment=102882721242&sc_matchtype=e&sc_country=US&s_kwcid=AL!4422!3!102882721242!e!!g!!create%20an%20aws%20account&ef_id=Wd_k7wAAAVgVBk9m:20180604172833:s)
+
```

Comment:
I'd add "Willingness to navigate around janky AWS UIs" here just to warn people that there be a few dragons. This feels higher level of difficulty than the rest of our tutorials -- do we want to say that this is a tutorial for developers that have a bit of experience and it may be challenging for newcomers?

Reply:
Sure, both great ideas. I'll do that right now then merge this!